code/modules/mechs/equipment/utility.dm

```diff
@@ -205,13 +205,27 @@
 
 /obj/item/material/drill_head/Initialize()
 	. = ..()
-	durability = 2 * (material ? material.integrity : 1)
+	// OCCULUS EDIT -- durability was not being properly applied because material during this proc is null
+	//durability = 2 * (material ? material.integrity : 1)
+
+/obj/item/material/drill_head/Created(var/creator) // PORTs another fix from Occulus
+	src.ApplyDurability()
+
+/obj/item/material/drill_head/steel/New(var/newloc)
+	..(newloc,MATERIAL_STEEL)
+	src.ApplyDurability()	// OCCULUS EDIT -- apply durability to drills properly
 
 /obj/item/material/drill_head/plasteel/New(var/newloc)
 	..(newloc,MATERIAL_PLASTEEL)
+	src.ApplyDurability()	// OCCULUS EDIT -- apply durability to drills properly
 
 /obj/item/material/drill_head/diamond/New(var/newloc)
 	..(newloc,MATERIAL_DIAMOND)
+	src.ApplyDurability()	// OCCULUS EDIT -- apply durability to drills properly
+
+///// OCCULUS EDIT -- handy verb to prevent copy/pasting durability in each New proc
+/obj/item/material/drill_head/verb/ApplyDurability()
+	durability = 2 * (material ? material.integrity : 1)
```

Comment:
Doesn't need occulus edit as it's on eris now, you can also remove the `.src` from code.

Reply:
got rid of the occulus stuff, what'dya mean the .src's? I am not a coder

Reply:
Got rid of the src's on the areas you mentioned